app/models/manageiq/providers/amazon/network_manager/refresher.rb

```diff
@@ -3,7 +3,11 @@ class Amazon::NetworkManager::Refresher < ManageIQ::Providers::BaseManager::Refr
     include ::EmsRefresh::Refreshers::EmsRefresherMixin
 
     def parse_legacy_inventory(ems)
-      ManageIQ::Providers::Amazon::NetworkManager::RefreshParser.ems_inv_to_hashes(ems, refresher_options)
+      if refresher_options.try(:[], :dto_refresh)
```

Comment:
Can you ask the EMS if it `supports? :dto_refresh`?  Or, is the EMS the wrong layer to ask that?

I'd like to see if there's a well defined way of denoting this support other than packing it into options.

#future #pivotal